keck_vnc_launcher.py

```diff
@@ -618,13 +632,13 @@ def launch_vncviewer(self, vncserver, port, geometry=None):
         vncargs = self.config.get('vncargs', None)
 
         cmd = [vncviewercmd]
-        if vncargs:
+        if vncargs is not None:
             vncargs = vncargs.split()
             cmd = cmd + vncargs
-        if self.args.viewonly:
+        if self.args.viewonly is not None:
```

Comment:
This caused ViewOnly to always activate because self.args.viewonly is coming in as False by default, not None.

Reply:
Pushed the fix to this branch.